.travis.yml

```diff
@@ -61,6 +66,7 @@ before_script:
       bash tests/bin/install.sh woocommerce_test root '' localhost $WP_VERSION
       composer global require "phpunit/phpunit=5.7.*|7.5.*"
     fi
+  - "/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16"
```

Comment:
For future reference, could you please provide more details about what this command is doing and why it is needed? Thanks.

Reply:
Will add a comment